### PR TITLE
Gameplay and tests

### LINKED ISSUE #1: US-2: As a player, I want to be able to start a new game from scratch

### LINKED ISSUE #2: US-3: As a player, I want to be able to save my current game so that I can continue at a later time

### LINKED ISSUE #6: US-4: As a player, I want to be able to exit the main menu
by @jordie007

### LINKED ISSUE #19: US-6b: Build Beach

### PR DESCRIPTION
**Pull request by CaeNGnp**
Resolves #1 
Resolves #6 
Resolves #19

### CODE REVIEW

File: SimpCityTests/GameTests.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+namespace SimpCityTests {
+    /// <summary>
+    /// Tests the gameplay
+    /// </summary>
+    [TestClass]
+    public class GameTests {
+        /// <summary>
+        /// QA-SN-5.1:
+        /// This ensures that the game can place buildings down.
```
Comment (CaeNGnp) [inline SimpCityTests/GameTests.cs:13]: e.g.

**Comment by CaeNGnp** [comment]
QA; Please also take note of the unit tests that are labelled corresponding to the test cases written in Google sheets. These are the ones we discussed that can be automated.

Those labelled `N/A` are additional developer unit tests which are not directly related to the test cases.

**Review by jordie007 — APPROVED** [review]
The gameplay and test codes are good to go already.

**Review by JSQJ — APPROVED** [review]
Test no issues, code works fines.

**Pull request merged**